src/utils/daos_hdlr.c

```diff
@@ -2831,9 +2845,10 @@ fs_copy_hdlr(struct cmd_args_s *ap)
 	struct duns_attr_t	dst_dattr = {0};
 	struct file_dfs		src_file_dfs = {0};
 	struct file_dfs		dst_file_dfs = {0};
-	struct fs_copy_args	fa = {0};
+	struct copy_args	fa = {0};
```

Comment:
@dsikich Minor nitpick:
Since we changed `fs_copy_args` to `copy_args`, maybe we should also change `fa` to `ca` or something similar?

Reply:
@daltonbohning I wasn't really looking at it that way. I kept the name since that instance of it is the "filesystem args," but I can go ahead and update it, doesn't really matter to me either way.

Reply:
@dsikich That's okay, certainly not a big deal.